fotoapparat/src/main/java/io/fotoapparat/parameter/selector/PreviewFpsRangeSelectors.java

```diff
@@ -0,0 +1,77 @@
+package io.fotoapparat.parameter.selector;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Comparator;
+
+import io.fotoapparat.parameter.range.Range;
+import io.fotoapparat.util.CompareFpsRangeByBounds;
+import io.fotoapparat.util.ExactFpsRangePredicate;
+import io.fotoapparat.util.InBoundsFpsRangePredicate;
+
+import static io.fotoapparat.parameter.selector.Selectors.filtered;
+import static io.fotoapparat.parameter.selector.Selectors.firstAvailable;
+
+/**
+ * Selector functions for preview FPS ranges.
+ */
+public class PreviewFpsRangeSelectors {
+
+    private static final Comparator<Range<Integer>> COMPARATOR_BY_BOUNDS = new CompareFpsRangeByBounds();
+
+    /**
+     * For conversion fps to FPS range bound
+     */
+    private static final int FPS_RANGE_BOUNDS_SCALE = 1000;
+
+    /**
+     * @param fps the specified FPS
+     * @return {@link SelectorFunction} which selects FPS range that contains only the specified FPS.
+     */
+    public static SelectorFunction<Range<Integer>> fromExactFps(Integer fps) {
+        return filtered(biggestFpsRange(),
+                new ExactFpsRangePredicate(fps * FPS_RANGE_BOUNDS_SCALE));
+    }
+
+    /**
+     * @param fps the specified FPS
+     * @return {@link SelectorFunction} which selects FPS range that contains the specified FPS.
+     */
+    public static SelectorFunction<Range<Integer>> nearestToExactFps(Integer fps) {
+        return firstAvailable(fromExactFps(fps),
+                filtered(biggestFpsRange(), new InBoundsFpsRangePredicate(fps * FPS_RANGE_BOUNDS_SCALE)));
+    }
+
+    /**
+     * @return {@link SelectorFunction} which selects FPS range with max FPS.
+     */
+    public static SelectorFunction<Range<Integer>> biggestFpsRange() {
```

Comment:
For me ```widestRange``` intends range with highest difference between upper and lower bounds. Confusing :(
Alternatively, I can suggest rename this method to ```rangeWithHighestFps```.